planner/core/pb_to_plan.go

```diff
@@ -0,0 +1,193 @@
+package core
```

Comment:
Please add a license header for this file.

Reply:
It's better to change the filename from `pb_to_plan` to `pb_plan_builder` to align the file content.

Reply:
The file of `pb_to_plan.go` is associated with `plan_to_pb.go`. So I prefer `pb_to_plan`

Reply:
OK

Reply:
@crazycs520 License header should be added to this file too.